lib/content_change_email_status_count.rb

```diff
@@ -0,0 +1,47 @@
+class ContentChangeEmailStatusCount
```

Comment:
Maybe we can put this in a `lib/reports` module to future reports together. We could then also move the `UnpublishingReport` into that folder.